dwave/system/composites/reversecomposite.py

```diff
@@ -228,13 +228,47 @@ def parameters(self):
     def properties(self):
         return {'child_properties': self.child.properties.copy()}
 
-    def sample(self, bqm, **parameters):
+    def sample(self, bqm, initial_states=None, initial_states_generator='random', num_reads=None, 
+               seed=None, **parameters):
         """Sample the binary quadratic model using reverse annealing from multiple initial states.
 
         Args:
             bqm (:obj:`dimod.BinaryQuadraticModel`):
                 Binary quadratic model to be sampled from.
 
+            initial_states (samples-like, optional, default=None):
+                One or more samples, each defining an initial state for all the problem variables. 
+                If fewer than `num_reads` initial states are defined, additional values are 
+                generated as specified by `initial_states_generator`. See :func:`dimod.as_samples` 
+                for a description of "samples-like".
+
+            initial_states_generator ({'none', 'tile', 'random'}, optional, default='random'):
+                Defines the expansion of `initial_states` if fewer than
+                `num_reads` are specified:
+
+                * "none":
+                    If the number of initial states specified is smaller than
+                    `num_reads`, raises ValueError.
+
+                * "tile":
+                    Reuses the specified initial states if fewer than `num_reads`
+                    or truncates if greater.
+
+                * "random":
+                    Expands the specified initial states with randomly generated
+                    states if fewer than `num_reads` or truncates if greater.
+
+            num_reads (int, optional, default=len(initial_states) or 1):
```

Comment:
Right now, `num_reads` is being consumed by this composite and won't be propagated to the underlying sampler

Reply:
I suppose because you're sending one read for each parsed initial state?

Reply:
Yes, that’s what I was thinking. If we send num_reads to the underlying sampler then it’ll be num_reads for each initial state. I'm not sure what the expected behaviour would be here

Reply:
One thing that we could do to save on network calls is [.aggregate](https://docs.ocean.dwavesys.com/en/stable/docs_dimod/reference/generated/dimod.SampleSet.aggregate.html#dimod.SampleSet.aggregate) the parsed sampleset and then have `num_reads` match the `num_occurrances`. Of course this would only work when the child sampler accepts `num_reads`.